examples/client/getBlockInfo.ts

```diff
@@ -0,0 +1,56 @@
+import { BlockInfo } from '@concordium/common-sdk';
+import { createConcordiumClient } from '@concordium/node-sdk';
+import { credentials } from '@grpc/grpc-js';
+
+import meow from 'meow';
+
+const cli = meow(
+    `
+  Usage
+    $ yarn ts-node <path-to-this-file> [options]
+
+  Options
+    --help,     -h  Displays this message
+    --block,    -b  A block to query from, defaults to last final block
+    --endpoint, -e  Specify endpoint of the form "address:port", defaults to localhost:20000
+`,
+    {
+        importMeta: import.meta,
+        flags: {
+            endpoint: {
+                type: 'string',
+                alias: 'e',
+                default: 'localhost:20000',
+            },
+            block: {
+                type: 'string',
+                alias: 'b',
+                default: '', // This defaults to LastFinal
+            },
+        },
+    }
+);
+
+const [address, port] = cli.flags.endpoint.split(':');
+const client = createConcordiumClient(
+    address,
+    Number(port),
+    credentials.createInsecure(),
+    { timeout: 15000 }
+);
+
+if (cli.flags.h) {
+    cli.showHelp();
+}
+
+/// Retrieves information about a specific block.
+/// If a blockhash is not supplied it will pick the latest finalized block.
+
+(async () => {
+    const blockInfo: BlockInfo = await client.getBlockInfo(cli.flags.block);
+
+    console.dir(blockInfo, { depth: null, colors: true });
+
+    // The blockInfo contain information that can then be extracted:
```

Comment:
We could do a bit more effort, such as printing "a block with N transactions baked at ?. It's parent is ...". Or some such.

Reply:
I could do a few of those. Do we then want to keep line 52 that prints the whole block then?

The direction I've been going so far is to print the entire returned object, and if applicable, extract information as unused `const`'s that are not printed. Do we want to print things like this or do I continue defining them as unused `const`'s but with better documentation/comments?

Reply:
It's up to you, but I would go for something like printing a few interesting things only. Dumping large objects is not super nice. They should get intellisense support in VSCode I assume when modifying the example.